### PR TITLE
#455 Use LineStylesWithColor instead of SingleColor

### PR DESCRIPTION
**Pull request by langonginc**

### CODE REVIEW

File: src/util/change-types.ts [hunk 1]
```diff
@@ -124,7 +124,6 @@ export const changeLineStyleType = (
     const newAttrs = structuredClone(lineStyles[newLineStyleType].defaultAttrs);
     if (LineStylesWithColor.includes(currentLineStyleType) && LineStylesWithColor.includes(newLineStyleType))
         (newAttrs as AttributesWithColor).color = (oldAttrs as AttributesWithColor).color;
-    else if (newLineStyleType === LineStyleType.SingleColor && theme) (newAttrs as SingleColorAttributes).color = theme;
```
Comment (thekingofcity) [inline src/util/change-types.ts:127]: Hi, I believe this is used when the user is changing the style to the single color style and the current theme will be applied automatically.

Comment (langonginc) [reply src/util/change-types.ts:127]: L126 has done it 👀

Comment (langonginc) [reply src/util/change-types.ts:127]: Oh, It's my fault. I'll put it back.

**Review by thekingofcity — APPROVED** [review]
LGTM

**Pull request merged**